src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
```

Comment:
Adding the "write-behind" object sets the store to be async, which most users won't want as the default behaviour. We should remove this.